src/System.Net.HttpListener/src/PinvokeAnalyzerExceptionList.analyzerdata

```diff
@@ -0,0 +1,46 @@
+sspicli.dll!EncryptMessage
```

Comment:
in commit efe2fc3 I'm ordering and removing dupes of all of these baseline files.